CMakeLists.txt

```diff
@@ -37,4 +37,4 @@ add_executable(tmatrix ${tmatrix_SRC})
 # Specify the include directory.
 include_directories(include)
 
-install(TARGETS tmatrix DESTINATION bin)
+install(TARGETS tmatrix DESTINATION usr/bin)
```

Comment:
This is not actually how CMake [install()](https://cmake.org/cmake/help/latest/command/install.html) works. The `bin` here is appended to the [`CMAKE_INSTALL_PREFIX`](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html) variable which is `/usr/local` by default. This is why in the README.md its instructed to invoke cmake with `-DCMAKE_INSTALL_PREFIX:PATH=bin/` which changes this variable externally. And I just noticed that is should be `/usr` not `bin/` :man_facepalming: , I'll change that right away.
Anyway, there is a right way to change the variable internally without also overriding user choice, you can read about [here](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT.html).

Reply:
Indeed you are right. That `bin/` in the instructions did confuse me.